core/src/main/java/org/apache/accumulo/core/client/impl/thrift/TableOperationExceptionType.java

```diff
@@ -30,11 +30,12 @@
   OFFLINE(2),
   BULK_BAD_INPUT_DIRECTORY(3),
   BULK_BAD_ERROR_DIRECTORY(4),
-  BAD_RANGE(5),
-  OTHER(6),
-  NAMESPACE_EXISTS(7),
-  NAMESPACE_NOTFOUND(8),
-  INVALID_NAME(9);
+  BULK_BAD_LOAD_MAPPING(5),
+  BAD_RANGE(6),
+  OTHER(7),
+  NAMESPACE_EXISTS(8),
+  NAMESPACE_NOTFOUND(9),
+  INVALID_NAME(10);
```

Comment:
Need to put the new type at the END of the list so the existing numbers don't change.